src/platform/silabs/wifi/wf200/WifiInterfaceImpl.cpp

```diff
@@ -367,6 +365,73 @@ bool IsStationConnected()
     return wifi_extra.Has(WifiState::kStationConnected);
 }
 
+bool IsHardwareReady()
+{
+    return wifi_extra.Has(WifiState::kStationInit);
+}
+
+CHIP_ERROR TriggerWifiDisconnection(void)
```

Comment:
only `disconnect` would be better under the abstraction namespace since we have moved to cpp right ?

Reply:
renamed the function